financial-connections/src/main/java/com/stripe/android/financialconnections/features/error/ErrorViewModel.kt

```diff
@@ -49,9 +50,14 @@ internal class ErrorViewModel @Inject constructor(
     }
 
     override fun updateTopAppBar(state: ErrorState): TopAppBarStateUpdate {
+        // Either the error that we correctly retrieved from the repository, or the error
+        // that occurred during retrieval.
+        val error = state.payload()?.error ?: state.payload.error
```

Comment:
This is a bit ridiculous now. We’ll leave this here until we rethink how we display errors (meaning if we move them back to a `LoadableContent` in every pane).